.github/workflows/publish.yml

```diff
@@ -41,9 +41,15 @@ jobs:
           java-version: '17'
           distribution: 'temurin'
           cache: 'gradle'
-      - name: Publish Library Artifactory
-        run: ./gradlew --build-cache artifactoryPublish
+      - name: Authenticate to GCP
+        uses: google-github-actions/auth@v2
+        with:
+          token_format: access_token
+          workload_identity_provider: 'projects/1038484894585/locations/global/workloadIdentityPools/github-wi-pool/providers/github-wi-provider'
+          service_account: 'dsp-artifact-registry-push@dsp-artifact-registry.iam.gserviceaccount.com'
+      - name: Publish client to GAR
+        run: ./gradlew publish
         env:
-          ARTIFACTORY_USERNAME: ${{ secrets.ARTIFACTORY_USERNAME }}
-          ARTIFACTORY_PASSWORD: ${{ secrets.ARTIFACTORY_PASSWORD }}
-          ARTIFACTORY_REPO_KEY: "libs-release-local"
```

Comment:
does this mean we can also remove https://github.com/broadinstitute/terraform-ap-deployments/blob/824166e7be8cfa8be5c4954a07c1aa5a9fcab894/github/tfvars/databiosphere-java-pfb.tfvars#L22-L31 ? It might make sense to do a big bulk update in terraform-ap-deployments once we've changed over multiple repos.

Reply:
Probably, maybe a bulk update is a good idea though.  But this comment did bring my attention to directions in the README that need to be updated, thank you!